src/routes/breaddit/breaddit.controller.js

```diff
@@ -0,0 +1,104 @@
+const fs = require("fs");
+const { promisify } = require("util");
+const { validationResult } = require("express-validator");
+
+const breadditsData = require("../../../db/dummy.data.json");
+const data = breadditsData.breaddits;
+
+const writeFile = promisify(fs.writeFile);
+
+const listBreaddits = (req, res) => {
+  return res.json({
+    data
+  });
+};
+
+const createBreaddit = async (req, res) => {
+  // console.log('body', req.body)
+  const id = data.length + 100;
```

Comment:
This was interesting, why `+ 100`?

Reply:
I was playing around with express-validators checking on number values.